src/main/java/com/cachinginapp/enterprise/dto/Cache.java

```diff
@@ -5,8 +5,14 @@
 public @Data
 class Cache {
     private int cacheID;
-    private String latitude;
-    private String longitude;
+    private int latitude;
+    private int longitude;
```

Comment:
I would not use an int to represent lat/lng.  The part after the decimal is important, and ints only store whole numbers.